### PR TITLE
Fixes #629

### LINKED ISSUE #629: [BUG] Unfired Pot missing turntable recipe
**Describe the bug**
The unfired pot cannot be turned into an unfired vase on the turntable.

**To Reproduce**
Place a clay block onto a powered turntable from better with mods.
Block will turn into an unfired planter, then an unfired pot. The crafting will halt there due to the missing recipe.

**Expected behavior**
The pot should be turned into a vase after a moment which can then turn into an urn.

**Screenshots**
[gyazo screenshot](https://i.gyazo.com/19dbaa3bdaf36c95bee74497201022b8.png)

**Info (please complete the following information):**
 - Modpack Version: 3.0 Beta v3
 - Server or Singleplayer: Server (HydroX)
 - Version the world was created on: 3.0 Beta v3

### PR DESCRIPTION
**Pull request by TheNelston**
Fixes the following issues:
- Unfired vase -> Unfired pot recipe fixed (no longer requires fired version + removed clay ball drop for balance)
- Added Unfired urn -> Clay ball recipe

### CODE REVIEW

**Comment by democat3457** [comment]
but what about the rustic vase

**Comment by TheNelston** [comment]
Rustic vase was causing the issue, the broken recipe was rustic vase (the fired version) to unfired pot, which makes no sense

Also don't remove the air, testing shows that it's buggy for some reason and causes the urn to never break but continuously produce clay like a duping machine

File: scripts/betterwithmods/betterwithmods.zs [hunk 1]
```diff
@@ -111,7 +111,7 @@ Turntable.add(<minecraft:clay>, <betterwithmods:unfired_pottery:1>, [<minecraft:
 Turntable.add(<betterwithmods:unfired_pottery:1>, <contenttweaker:pot_unfired>, []);
 Turntable.add(<contenttweaker:pot_unfired>, <betterwithmods:unfired_pottery:3>, [<minecraft:clay_ball>]);
 Turntable.add(<betterwithmods:unfired_pottery:3>, <betterwithmods:unfired_pottery:2>, [<minecraft:clay_ball>]);
-Turntable.add(<betterwithmods:unfired_pottery:2>, <minecraft:air>, [<minecraft:clay_ball>]);
+Turntable.add(<betterwithmods:unfired_pottery:2>, [<minecraft:clay_ball>]);
```
Comment (TheNelston) [inline scripts/betterwithmods/betterwithmods.zs:114]: add back air

**Review by TheNelston — COMMENTED** [review]
add back the air

File: scripts/betterwithmods/betterwithmods.zs (hunk 1 above)
Comment (democat3457) [reply scripts/betterwithmods/betterwithmods.zs:114]: wtf thats wack

Comment (democat3457) [reply scripts/betterwithmods/betterwithmods.zs:114]: ya even the code basically replaces the missing argument with air anyway, so it doesn't seem like it should matter

Comment (TheNelston) [reply scripts/betterwithmods/betterwithmods.zs]: idk man shits broken for me in game

**Review by democat3457 — APPROVED** [review]

**Review by Foreck1 — APPROVED** [review]

**Pull request merged**